swiftpkg/internal/swiftpkg_build_files.bzl

```diff
@@ -49,6 +49,7 @@ def _swift_target_build_file(pkg_ctx, target):
     attrs = {
         "deps": bzl_selects.to_starlark(deps),
         "module_name": target.c99name,
+        "package_name": target.label.repository_name.lstrip("@") + ".rspm",
```

Comment:
Is it possible to set this to something custom in SPM today? I know it is in Xcode via `PACKAGE_ACCESS_IDENTIFIER`. Just wondering if we would ever run into issues where one package defines `package` visible APIs that are then used by another package.

Reply:
> Is it possible to set this to something custom in SPM today? I know it is in Xcode via `PACKAGE_ACCESS_IDENTIFIER`. Just wondering if we would ever run into issues where one package defines `package` visible APIs that are then used by another package.

No, not via SwiftPM; it uses the package name from Package.swift.